snippets/dart_snippets/lib/closed_channel.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:breez_sdk/breez_sdk.dart';
+import 'package:breez_sdk/bridge_generated.dart';
+
+Future<PrepareSweepResponse> prepareSweep({required int feeRate}) async {
+  // ANCHOR: prepare-sweep
+  final satPerVbyte = feeRate;
```

Comment:
```suggestion
Future<PrepareSweepResponse> prepareSweep({required int satPerVbyte}) async {
  // ANCHOR: prepare-sweep
```

Reply:
I did not add it here for consistency with the other examples.

Reply:
@ubbabeck Where are the other Dart examples that use this format? There should be a mistake.